spec/vcr_cassettes/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher.yml

```diff
@@ -5,81 +5,88 @@ http_interactions:
         openid"}'
     http_version:
-  recorded_at: Wed, 14 Oct 2020 14:29:40 GMT
+  recorded_at: Wed, 17 Feb 2021 18:03:12 GMT
 - request:
     method: get
-    uri: https://us-east.iaas.cloud.ibm.com/v1/floating_ips?generation=2&version=2020-08-01
+    uri: https://us-east.iaas.cloud.ibm.com/v1/floating_ips?generation=2&version=2021-01-01
     body:
-      encoding: US-ASCII
+      encoding: UTF-8
       string: ''
     headers:
-      Authorization:
-      - Bearer eyJraWQiOiIyMDIwMDkyMjE4MzMiLCJhbGciOiJSUzI1NiJ9.eyJpYW1faWQiOiJJQk1pZC0wNjAwMDA0UzM4IiwiaWQiOiJJQk1pZC0wNjAwMDA0UzM4IiwicmVhbG1pZCI6IklCTWlkIiwianRpIjoiYjQ2MDE0YmQtNTdiZS00ZjE5LWJjMGMtYzBjZjM0YzkxMmZjIiwiaWRlbnRpZmllciI6IjA2MDAwMDRTMzgiLCJnaXZlbl9uYW1lIjoiQnJhZCIsImZhbWlseV9uYW1lIjoiQmxhbmNldHQiLCJuYW1lIjoiQnJhZCBCbGFuY2V0dCIsImVtYWlsIjoiYmxhbmNldHRAdXMuaWJtLmNvbSIsInN1YiI6ImJsYW5jZXR0QHVzLmlibS5jb20iLCJhY2NvdW50Ijp7InZhbGlkIjp0cnVlLCJic3MiOiJjNTZjOWEyNjhkMjNlMWIzMzlhYzE0Nzc0MzU4MTMzYyIsImltc191c2VyX2lkIjoiNzI4MjEwMyIsImZyb3plbiI6dHJ1ZSwiaW1zIjoiMTE2MDQ0NyJ9LCJpYXQiOjE2MDI2ODU3NzcsImV4cCI6MTYwMjY4OTM3NywiaXNzIjoiaHR0cHM6Ly9pYW0uY2xvdWQuaWJtLmNvbS9pZGVudGl0eSIsImdyYW50X3R5cGUiOiJ1cm46aWJtOnBhcmFtczpvYXV0aDpncmFudC10eXBlOmFwaWtleSIsInNjb3BlIjoiaWJtIG9wZW5pZCIsImNsaWVudF9pZCI6ImRlZmF1bHQiLCJhY3IiOjEsImFtciI6WyJwd2QiXX0.cUE-ax1vk0g_E5NM9eJYXVSKSxQwt4ajcK8xRpiUflJomzNgiqRXmWraRCH-G4qHr5QpMxEhNZx8oZjOgsUoIdmFkhei7bpV12aH0NUuBJ8lGvjgt5X_V6zOD_OdKVwyc8TKATSeso25_By8eHxp9hmvrO0p09Q59h6PiITnqdDf_oiZGQvKj807qzfud68urei7FZbqBCt4KQnVhDYBH_1jiVCYKvpB6IcVuvcCBgHINyuqlUYxM7R86L62qM3aqKJKjja6IlKfr_PAcH5ajfBS097ogiUkqDA4wAETYuxBVWoq-Pd9FaOjIyHP9GgFNNWjCsB8qMhd6qUTapPVRQ
-      Accept-Encoding:
-      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
-      Accept:
-      - "*/*"
       User-Agent:
-      - Ruby
+      - ibm_vpc-ruby-sdk-0.1.0 x86_64-apple-darwin19.6.0 ruby-2.6.6
+      X-Ibmcloud-Sdk-Analytics:
+      - service_name=vpc;service_version=V1;operation_id=list_floating_ips
+      Accept:
+      - application/json
+      Authorization: Bearer xxxxxx
```

Comment:
I noticed that before, this value was an Array and now it's a scalar.  Is that expected?  Not sure if that's a side effect of the sanitization, and whether or not it's a problem or not.

Reply:
It is a product of the sanitization. It is okay. Spec tests pass because they aren't submitting a live request and thus don't care about authorization.

Reply:
If we're the ones doing the sanitization we might as well have it match what the request would look like

Reply:
Why? Once the rest call has been authenticated and returned its payload this header is useless.